Titanium.Web.Proxy/Network/Certificate/BCCertificateMaker.cs

```diff
@@ -146,7 +149,7 @@ public X509Certificate2 MakeCertificate(string sSubjectCn, bool isRoot, X509Cert
             {
                 try
                 {
-                    x509Certificate.FriendlyName = subjectName;
+                    x509Certificate.FriendlyName = System.Text.RegularExpressions.Regex.Replace(subjectName.ToLower(), @"^" + "CN".ToLower() + @"\s*=\s*", "");
```

Comment:
Why don't you write simply 
@"^cn\s*=\s*"
?

And this regex sshould be compiled.